x-pack/test/apm_api_integration/tests/services/throughput.ts

```diff
@@ -6,21 +6,21 @@
  */
 
 import expect from '@kbn/expect';
-import { first, last, mean } from 'lodash';
+import { first, last, mean, meanBy } from 'lodash';
 import moment from 'moment';
 import { isFiniteNumber } from '../../../../plugins/apm/common/utils/is_finite_number';
 import { APIReturnType } from '../../../../plugins/apm/public/services/rest/createCallApmApi';
+import { createApmApiSupertest } from '../../common/apm_api_supertest';
 import archives_metadata from '../../common/fixtures/es_archiver/archives_metadata';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
 import { registry } from '../../common/registry';
-import { createApmApiSupertest } from '../../common/apm_api_supertest';
 
 type ThroughputReturn = APIReturnType<'GET /api/apm/services/{serviceName}/throughput'>;
 
 export default function ApiTest({ getService }: FtrProviderContext) {
   const apmApiSupertest = createApmApiSupertest(getService('supertest'));
```

Comment:
This looks like the old syntax. 

```ts
const apmApiClient = getService('apmApiClient');
```

and then

```ts
const response = await apmApiClient.readUser({...})
```

Reply:
This is based on `7.15` that is why the old syntax is still being used.

Reply:
ah, okay. I'm not sure it's worth backporting to 7.15 at this point tbh.